content/en/docs/started/installing-kubeflow.md

```diff
@@ -229,17 +242,26 @@ The following table lists <b>legacy distributions</b> which have <b>not had a re
 <a id="install-the-kubeflow-manifests-manually"></a>
 ## Raw Kubeflow Manifests
 
+The raw Kubeflow Manifests are aggregated by the [Manifests Working Group](https://github.com/kubeflow/community/tree/master/wg-manifests)
+and are intended to be used as the __base of packaged distributions__.
+
+Very advanced users may choose to install the manifests for a specific Kubeflow version by following the instructions in the `README` of the [`kubeflow/manifests`](https://github.com/kubeflow/manifests) repository.
+
+- [__Kubeflow 1.8:__](/docs/releases/kubeflow-1.8/)
+   - [`v1.8-branch`](https://github.com/kubeflow/manifests/tree/v1.8-branch#installation) <sup>(development branch)</sup>
+   - [`v1.8.0`](https://github.com/kubeflow/manifests/tree/v1.8.0#installation)
+- [__Kubeflow 1.7:__](/docs/releases/kubeflow-1.7/)
+   - [`v1.7-branch`](https://github.com/kubeflow/manifests/tree/v1.7-branch#installation) <sup>(development branch)</sup>
+   - [`v1.7.0`](https://github.com/kubeflow/manifests/tree/v1.7.0#installation)
+
 {{% alert title="Warning" color="warning" %}}
-This method is for advanced users.
+Kubeflow is a complex system with many components and dependencies.
+Using the raw manifests requires a deep understanding of Kubernetes, Istio, and Kubeflow itself.
 
-The Kubeflow community is not able to provide support for environment-specific issues when using the raw manifests.
+When using the raw manifests, the Kubeflow community is not able to provide support for environment-specific issues or custom configurations.
```

Comment:
What about

"When using the raw manifests, the Kubeflow maintainers will not provide unpaid support for environment-specific issues or custom configurations. Nevertheless we welcome contributions and bug reports very much."

instead of

"When using the raw manifests, the Kubeflow community is not able to provide support for environment-specific issues or custom configurations."

Reply:
@juliusvonkohout Can you submit followup PR to make this change ?